src/store/match/store.ts

```diff
@@ -22,8 +22,7 @@ export const useMatchStore = defineStore("match", {
     selectedSeason: {} as Season,
   }),
   actions: {
-    async loadMatches(page?: number) {
```

Comment:
This gets called from more places than Matches.vue, for example PlayerMatchesTab.

Reply:
I tried checking other uses for this, but couldn't find any. The one you mention seems to be using a similar function, but in the `playerStore` and not `matchStore`